mainsite/selling/views.py

```diff
@@ -34,8 +34,12 @@ def index(request):
                 item_price = form.cleaned_data['item_price']
                 item_title = form.cleaned_data['item_title']
                 username = request.user
+                first_name = request.user.get_short_name()
+                # Use username if first_name is empty (null)
+                if not first_name:
```

Comment:
This block of code could be removed later when we do data validation in the account page. For now, if the user does not have a first name (such as users already in the database without a first name), then we will display their username. If we can guarantee users cannot change their first name to "", then we no longer need this preventative measure (though it doesn't hurt anything).